app/moves/index.js

```diff
@@ -43,7 +43,7 @@ router.get(
 router.get(
   '/:date/download.:extension(csv|json)',
   protectRoute('moves:download:all'),
-  setMovesByDateAndLocation,
+  setMovesByDateAllLocations,
```

Comment:
I know this is an emergency fix, but shouldn't we be writing a failing test? If we had had a test for this scenario originally, then the regression might not have happened?